receiver/sqlserverreceiver/scraper.go

```diff
@@ -168,6 +168,7 @@ func (s *sqlServerScraperHelper) recordDatabaseIOMetrics(ctx context.Context) er
 		rb.SetSqlserverInstanceName(row[instanceNameKey])
 		rb.SetServerAddress(s.config.Server)
 		rb.SetServerPort(int64(s.config.Port))
+		rb.SetHostName(s.config.Server)
```

Comment:
After this PR, the `server.address` and `host.name` resource attributes will always have the same value. Is there a benefit to this approach, verse simply removing documentation for the `host.name` resource attribute?

Reply:
Good question. `server.address` and `server.port` are actually attributes. They are not resource attributes, we should remove these two from resource. https://opentelemetry.io/docs/specs/semconv/attributes-registry/server/#server-address

While `host.name` is a resource attribute. https://opentelemetry.io/docs/specs/semconv/attributes-registry/host/#host-name

Reply:
should i remove those in this pr?

Reply:
updated

Reply:
Thanks for sharing the links @XSAM! Pardon my ignorance, but I'm not seeing how those links show that `server.address` and `server.port` are attributes rather than resource attributes. Can you show me why that's the case?

Reply:
updated